tally_ho/apps/tally/static/js/data_table.js

```diff
@@ -1,19 +1,40 @@
-$(document).ready(function() {
-    var dt_table = $('.datatable').dataTable({
-        language: dt_language,  // global variable defined in html
-        order: [[ 0, "desc" ]],
-        lengthMenu: [[10, 25, 50, 100, 500], [10, 25, 50, 100, 500]],
-        columnDefs: [
-            {orderable: true,
-             searchable: true,
-             className: "center",
-             targets: [0, 1]
-            }
-        ],
-        searching: true,
-        processing: true,
-        serverSide: true,
-        stateSave: true,
-        ajax: LIST_JSON_URL
-    });
+$(document).ready(function () {
+  $('.datatable').dataTable({
+    language: dt_language, // global variable defined in html
+    order: [[0, "desc"]],
+    lengthMenu: [
+      [10, 25, 50, 100, 500],
+      [10, 25, 50, 100, 500],
+    ],
+    columnDefs: [
+      {
+        orderable: true,
+        searchable: true,
+        className: "center",
+        targets: [0, 1],
+      },
+    ],
+    searching: true,
+    processing: true,
+    serverSide,
+    stateSave: true,
+    ajax: LIST_JSON_URL,
+    dom:
+      "<'row'<'col-sm-1'B><'col-sm-6'l><'col-sm-5'f>>" +
+      "<'row'<'col-sm-12'tr>>" +
+      "<'row'<'col-sm-5'i><'col-sm-7'p>>",
+    buttons: [
+      {
+        extend: "csv",
+        filename: exportFileName,
```

Comment:
not defined

-- if these are special flags for the library we can use override the style checks in-line

Reply:
These are special flags, I have ignore this file in the codacy config [here](https://github.com/onaio/tally-ho/blob/b854ebd638069b4843368cf217e1e500da24bfef/.codacy.yml#L4) and I can see it has been ignored in codacy [here](https://app.codacy.com/gh/onaio/tally-ho/settings/ignoreFiles) but it's still being captured.